rpc/rpcserver/server.go

```diff
@@ -60,6 +63,9 @@ const (
 	semverMajor  = 4
 	semverMinor  = 24
 	semverPatch  = 0
+
+	// copied from dcrd's rpc package
+	sstxCommitmentString = "sstxcommitment"
```

Comment:
wrong place to add this constant, and i find it dubious that we should be defining it here to begin with, especially without not defining any others